web/themes/custom/pineapple/composer.json

```diff
@@ -32,6 +32,11 @@
             "libraries/{$name}": [
                 "type:npm-asset"
             ]
+        },
+        "patches": {
+            "drupal/paragraphs": {
```

Comment:
Add this patch above the existing one 
```
"drupal/core": {
                "Media library widget + Paragraphs broken": "https://www.drupal.org/files/issues/2019-05-16/drupal-media-library-paragraphs-3003150-38-8.x.patch",
            },
```

Reply:
Run composer install to make sure it's aded

Reply:
done